### PR TITLE
Support sending files of quoted or escaped path

### PR DESCRIPTION
**Pull request by zhongzc**
Signed-off-by: Zhenchi <zhongzc_arch@outlook.com>

Can not send a file with spaces in its path. This PR will fix it.

Assumed I have a file:
```sh
 ➜  ~ touch 'a b'
```

with this patch I can send it via one of:
```
?send ~/a\ b
?send "~/a b"
?send "$HOME/a b"
?send $HOME/a\ b
...
```

### CODE REVIEW

**Comment by zhongzc** [comment]
Looking forward to your review! @lemunozm

File: src/commands/send_file.rs [hunk 1]
```diff
@@ -16,9 +16,12 @@ impl Command for SendFileCommand {
         "send"
     }
 
-    fn parse_params(&self, params: &[&str]) -> Result<Box<dyn Action>> {
-        let file_path = params.get(0).ok_or("No file specified")?;
-        match SendFile::new(file_path) {
+    fn parse_params(&self, param: Option<&str>) -> Result<Box<dyn Action>> {
+        let param = param.ok_or("No file specified")?;
+        let words = shellwords::split(param)?;
```
Comment (lemunozm) [inline src/commands/send_file.rs]: I like a lot the `shellwords` way of parsing commands :)

Only one point here. I think that this line and the previous one is something that any command should do. Could it be made before calling this function and pass the words from shellworlds directly as param of `parse_params`?

**Review by lemunozm — CHANGES_REQUESTED** [review]

File: src/commands/send_file.rs (hunk 1 above)
Comment (zhongzc) [reply src/commands/send_file.rs]: Updated. @lemunozm

**Review by lemunozm — APPROVED** [review]

**Comment by lemunozm** [comment]
Great! Thanks, @zhongzc for your contribution! 😃

**Pull request merged**